course6/toyapp_mikhail/displaylib/pyramid.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "matrix.h"
+#include "triangle.h"
+#include "surface3d.h"
+#include "rgb.h"
```

Comment:
You only need these includes here, because you have your implementation in this file.
And these includes propagate to every file that includes this one.

And then your compilation takes ages...